src/leetCodeExecutor.ts

```diff
@@ -34,7 +36,11 @@ class LeetCodeExecutor implements Disposable {
         return `"${path.join(this.leetCodeRootPath, "bin", "leetcode")}"`;
     }
 
-    public async meetRequirements(): Promise<boolean> {
+    public async meetRequirements(context: ExtensionContext): Promise<boolean> {
+        const isUserFresh: boolean | undefined = context.globalState.get(globalStateLeetcodeIsUserFresh);
+        if (isUserFresh !== false) {
```

Comment:
if (!isUserFresh)

Reply:
> if (!isUserFresh)

@jdneo 
Seems not the same logic if use if (!isUserFresh)